MapboxDirections/MBWaypoint.swift

```diff
@@ -106,6 +106,24 @@ open class Waypoint: NSObject, NSCopying, NSSecureCoding {
      */
     @objc open var coordinateAccuracy: CLLocationAccuracy = -1
     
+    internal enum CoordinatePrecision: Int {
+        case five = 5
+        case six = 6
+     }
+    
+    internal func jsonRepresentation(precision: CoordinatePrecision) -> String? {
```

Comment:
The return value goes directly into a URL query parameter, not JSON. What we’re implementing here is really `String(describing:precision:)` for a `CLLocationCoordinate2D`.